### DISCUSSION TITLE
New to GitHub Sponsor

### DISCUSSION THREAD

**Discussion by jonypeixoto** [community/community#64226] [Other Feature Feedback, Questions, & Ideas]
I need the Github sponsor, it's been weeks, but my application is still pending. I already have GitHub Sponsor in my main account and I want to open GitHub Sponsor in account whose user is:

@cybertimeup
https://github.com/cybertimeup

My app is still on the GitHub Sponsors waitlist!

Please help me

**Comment by xames3** [comment]
Are you sure you completed the profile correctly? I mean for me, it just got approved in like 7 hours after raising the request. Please make sure the request is complete and you provide all the details as to why your account/organization should be enrolled into the Sponsorship program. Can you please provide more information?

**Reply by jonypeixoto** [reply]
Yes, I provided all the necessary information. I already have a GitHub sponsor account activated, would that be the problem?

This activation would be my second activation on the GitHub Sponsor.

**Reply by jonypeixoto** [reply]
I have a screen print showing the message PENDING, analyzing the profile but I can never get approved. What can it be?

PRINT: https://ibb.co/GRTSLmX
